### PR TITLE
Allow attachments for tickets in sdf

### PR DESCRIPTION
**Pull request by PPelchmann**

### CODE REVIEW

**Comment by jane-doe** [comment]
Can one of the admins verify this patch?

**Review by dpolanowski — APPROVED** [review]

**Comment by KaushikGondaliya** [comment]
test

**Comment by PPelchmann** [comment]
test

File: SDF/Automation/change.json [hunk 1]
```diff
@@ -74,6 +81,11 @@
         "to": "comments",
         "from": "change",
         "id": "ogit/generates"
+      },
+      {
+        "to": "attachment",
```
Comment (Anusha-Arago) [inline SDF/Automation/change.json:86]: Should it be "to":"change" ?

**Comment by KaushikGondaliya** [comment]
test

File: SDF/Automation/change.json (hunk 1 above)
Comment (PPelchmann) [reply SDF/Automation/change.json]: 👍

**Review by dpolanowski — APPROVED** [review]

**Pull request merged**